src/index.ts

```diff
@@ -0,0 +1,104 @@
+import express from 'express';
```

Comment:
for now to keep thing simple, let's use the common js style for modules, which is supported out of the box by node.  so we will require our modules like this:
```suggestion
const express = require('express')
```
later on, if you are interested we can convert to typescript and use the es module syntax